source/includes/_webhooks.md

```diff
@@ -0,0 +1,380 @@
+
+## Payload Overview
+
+```json
+{
+    "organization_id": 1234,
+    "organization_name": "My organization",
+    "organization_owner_name": "Jane Doe",
+    "organization_owner_email": "support@electionbuddy.com",
+    "election_id": 12345,
+    "election_name": "My election",
+    "identifier": "1",
+    "email": "voter@example.com",
+    "sms": "555-5555",
+    "postal": "1, 3984 Massachusetts Avenue, Washington DC",
+    "status": [
+        "Voted",
+        "Key Surfaced"
+    ],
+    "opened_at": 1600000000,
+    "completed_at": 1600001234,
+    "added_at": null,
+    "spoiled_at": null,
+    "spoil_reason": null,
+    "key_surfaced_at": null,
+    "ballot_printed_at": null,
+    "ip": "0.0.0.0",
+    "unsubscribed_at": null
+    // Non-exhaustive.
+}
+```
+
+The HTTP POST requests will contain a JSON payload. The payload contains
+information about your vote, the organization associated with that vote,
+and the current status of the vote. The parts of the payload will be
+different depending on your vote configuration.
+
+Voter information, such as their postal address, SMS number and email address
+will be included if that information is included in the voter list.
+
+The `opened_at`, `completed_at`, etc. fields represent instants in time when the
+given event happened. If the gien event has happened then the value with be a
+UNIX timestamp. That is, a number of *seconds* after January 1st, 1970.
+If the given event has not happened, the value will be `null`.
+
+## Voter Choices Overview
+
+```json
+{
+  // ... other fields ...
+  "questions": [
+    {
+      "question": "Bylaw Amendment Approval of Article XLII",
+      "type": "plurality",
+      "answers": {
+        "regular": [
+          {
+            "title": "Yes - I approve the amendments",
+            "choice": "true"
+          }
+        ],
+        "write_ins": []
+      }
+    }
+  ]
+}
+```
+
+If your Voter Anonymity settings allow administrators to view voter choices,
+then voter choices will be included in the payload.
+
+![radio buttons offering three choices: Secret Ballot, Poll, and Show of Hands](voter-choices.png)
+
+There are six types of questions corresponding to the 6 voting system options
+available when adding Positions/Questions.
+
+![A drop down offering 6 choices: Plurality, Cumulative, Preferential, Approval, Nomination, and Scored](voting-system-options.png)
+
+Correspondingly, The `"type"` field on each object in the `"questions"` array
+will have one of the following values:
+
+* `"plurality"`
+* `"cumulative"`
+* `"preferential"`
+* `"approval"`
+* `"nomination"`
+* `"scored"`
+
+The `"answers"` field will have slightly different contents depending on the
+`"type"`. The overal shape will be the same but the `"choice"` fields,
+on the objects inside the `"regular"` and/or `"write_ins"` arrays, will be have
+different possible values and the proper way to interpret those values will be
+different.
```

Comment:
@Ryan1729 I think we have a leftover `be` here (but it's possible that you wanted `behave` here 😓):

```suggestion
on the objects inside the `"regular"` and/or `"write_ins"` arrays, will have
different possible values and the proper way to interpret those values will be
different.
```

Reply:
You are right that this was a leftover `be`, from a previous version of the sentence.